src/usdb_syncer/song_loader.py

```diff
@@ -290,6 +297,44 @@ def _write_sync_meta(ctx: Context) -> None:
     ctx.sync_meta.to_file(ctx.locations.dir_path)
 
 
+def _maybe_upload_txt(ctx: Context) -> None:
+    if not get_usdb_mod_status():
```

Comment:
Making an additional HTTP request for every user and every download seems excessive when it can't work for 99.9% of users anyway. I would use an environment variable instead. And if it's set, can't we just make the post request directly without checking the login status first?